correios/models/user.py

```diff
@@ -209,6 +209,10 @@ def __init__(self, number: int, code: str, name: str) -> None:
             raise InvalidExtraServiceError("Invalid Extra Service Name {!r}".format(name))
         self.name = name
 
+        if type(display_on_label) is not bool:
```

Comment:
Is it really necessary to check type here? I thought declaring type in parameter definition was enough.

Reply:
@cacarrara, I was looking the [PEP 484](https://www.python.org/dev/peps/pep-0484/) and says:
"While these annotations are available at runtime through the usual ___annotations___ attribute, no type checking happens at runtime. Instead, the proposal assumes the existence of a separate off-line type checker which users can run over their source code voluntarily."

Reply:
so, mypy does not work here ?

Reply:
Ah @georgeyk , I think I get it now. I will remove it.